apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -136,13 +136,13 @@ const findEditUrlData = [
   },
   {
     name: 'course admin via course instance',
-    selector: 'a:contains("infoCourse.json") + a:contains("Edit")',
+    selector: 'a:contains("Edit course configuration")',
     url: courseInstanceCourseAdminSettingsUrl,
     expectedEditUrl: courseInstanceCourseAdminEditUrl,
   },
   {
     name: 'course admin',
-    selector: 'a:contains("infoCourse.json") + a:contains("Edit")',
+    selector: 'a:contains("Edit course configuration")',
```

Comment:
I think relying on the button text is dangerous. Can you add a `data-test-id` argument (or similar) and select based on that?

Reply:
Good idea. I added the id `edit-course-configuration-link` to the link and added that id to the selector. Is this what you had in mind? 

`selector: '#edit-course-configuration-link, a:contains("Edit course configuration")',`

Reply:
You don't need both selectors in that case, only the ID. That said, using an ID only for testing may be a bit overkill, and may cause confusion since it's not used in production code. See @nwalters512's comment in https://github.com/PrairieLearn/PrairieLearn/pull/5793#discussion_r889198898.

Reply:
I see now. I have now added `data-testid`  and selecting by that instead of the id. 

The selector now looks like this: 
`selector: '[data-testid="edit-course-configuration-link"]',`